.buildkite/pipeline.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
+# Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+"""
```

Comment:
nit: Shall we put some comments here for this script?

Reply:
yep, the CI complained about it.